packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -96,20 +76,12 @@ describe('Ceramic API', () => {
   }
 
   beforeAll(async () => {
-    tmpFolder = await tmp.dir({ unsafeCleanup: true })
-
-    ipfs = await createIPFS({
-      repo: `${tmpFolder.path}`,
-      config: {
-        Addresses: { Swarm: [ `/ip4/127.0.0.1/tcp/${3001}` ] },
-        Bootstrap: []
-      }
-    })
+    ipfs = await createIPFS()
   })
 
   afterAll(async () => {
     await ipfs.stop(() => console.log('IPFS stopped'))
-    await tmpFolder.cleanup()
+    await ceramic?.close()
```

Comment:
Should we change this from an `afterAll` to an `afterEach`?  Otherwise if a test reassigns the `ceramic` variable without closing the existing one then the previous ceramic gets leaked

Reply:
Changed.